src/components/Register/RegisterFormComponent.tsx

```diff
@@ -0,0 +1,168 @@
+import React, {ChangeEvent, useState} from "react";
+import {Button, Divider, Form, Input, Checkbox, Upload} from "antd";
+import { UploadOutlined} from '@ant-design/icons';
+import {useTranslation} from "react-i18next";
+
+const RegisterFormComponent: React.FC = () => {
+    const {t} = useTranslation("register");
+    const [isValid, setIsValid] = useState<boolean>(false);
+    const [name, setName] = useState<string>();
+    const [surname, setSurname] = useState<string>();
+    const [nickname, setNickname] = useState<string>();
+    const [email, setEmail] = useState<string>();
+    const [password, setPassword] = useState<string>();
+    const [seePassword, setSeePassword] = useState<boolean>(false);
+
+
+
+    return (
+        <div className="bg-white m-4 md:m-10">
+            <div className="text-md hidden md:block md:text-lg w-full m-2 md:m-4 mb-6 text-center">{t("Registration")}</div>
+
+
+    <Form name="normal_register" className="register-form" >
+
+        <Form.Item>
+            <Upload >
+                <Button>
+                    <UploadOutlined /> Click to Upload a Profile Picture
+                </Button>
+            </Upload>
+        </Form.Item>
+
+        <Form.Item
+            name="name"
+            label="Name"
+            rules={[{ required: true, message: t('Please input your name!') }]}
+        >
+            <Input prefix={<i className="far fa-user text-gray-500 leading-8 mr-2"></i>}
+                   name="name"
+                   placeholder="Pierre"
```

Comment:
J'aurai mis `ex :` pour tous les placeholder
genre comme ca : 
```suggestion
                   placeholder="ex: Pierre"
```
Ca montre mieux que c'est un exemple, mais c'est mon point de vue à discuter